tests/parametric/test_crashtracking.py

```diff
@@ -15,6 +15,7 @@ class Test_Crashtracking:
     @missing_feature(context.library == "golang", reason="Not implemented")
     @missing_feature(context.library == "nodejs", reason="Not implemented")
     @missing_feature(context.library == "cpp", reason="Not implemented")
+    @missing_feature(context.library == "ruby", reason="Disabled due to instability of crash tracker APMLP-273")
```

Comment:
The reason makes me think it's a bug? if yes : 

```suggestion
    @bug(context.library == "ruby", reason="APMLP-273")
```